lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -161,6 +161,26 @@ def test_savefig_metadata(monkeypatch):
     }
 
 
+def test_invalid_metadata(monkeypatch):
```

Comment:
I don't think you need the `monkeypatch` fixture, right?

Reply:
Nope, copy&paste extras from the other test. Fixed.